gems/smithy-client/lib/smithy-client.rb

```diff
@@ -44,6 +44,14 @@
 # endpoints
 require_relative 'smithy-client/endpoint_rules'
 
+# codecs
+require_relative 'smithy-client/codec/cbor'
+
+# cbor
+require_relative 'smithy-client/cbor'
+require_relative 'smithy-client/cbor/encoder'
```

Comment:
cbor is already requiring these. Either remove these or remove them from cbor.rb

Reply:
Yeah - I dont have real preference on this for now, but long term I think this behavior depends on what we do for engines.